.github/dependabot.yml

```diff
@@ -9,4 +9,7 @@ updates:
     target-branch: main
     labels:
       - "npm"
-      - "dependencies"
\ No newline at end of file
+      - "dependencies"
+    commit-message:
+      prefix: "⬆️ "
```

Comment:
Hmm, when I run this locally, I get a complaint:

```shell
~/repos/learning-typescript-projects$ echo "⬆️ build(deps-dev): bump ts-jest from 28.0.0 to 28.0.2" | npx commitlint
⧗   input: ⬆️ build(deps-dev): bump ts-jest from 28.0.0 to 28.0.2
✖   Your commit should start with gitmoji code,please check the emoji code on https://gitmoji.dev/. [start-with-gitmoji]
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 3 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

Do you?

Reply:
I don't get this but, let me push a change in another format.. let me write it in gitmoji format instead of emoji.

Reply:
Can you try it now? I used gitmoji instead of emoji now.

Reply:
`echo ":arrow_up: build(deps-dev): bump abc to 123" | npx commitlint` works for me! 🚀 

I can't find a way to get Dependabot to create a _new_ pull request (do you know of one?), so I guess we have to ... wait until the next update comes in

Reply:
Yea we have to wait till another dependency update 😅